### ISSUE TITLE
Check for degraded DTLs

### ISSUE THREAD

**Issue by kinvaris** [openvstorage/openvstorage-health-check#100]
Check if a vdisk its DTL is degrated

**Comment by wimpers** [comment]
Please check for degra**d**ed DTL ;-)

**Comment by JeffreyDevloo** [comment]
Fixed by #129 , packaged in openvstorage-health-check_3.1.2-rev.207

**Comment by JeffreyDevloo** [comment]
### Steps
- Setup latest cluster
- configured latest cluster
- Tested for single cluster, multicluster

### Output
#### Single node
```
[WARNING] Vdisk myvdisk01's DTL is disabled because of a single node cluster
[INFO] Recap of volumedriver check_dtl!
[INFO] ======================
[INFO] SUCCESS=0 FAILED=0 SKIPPED=0 WARNING=1 EXCEPTION=0

```
#### Multicluster

```
root@ovs-node-1:~# ovs healthcheck volumedriver check_dtl
[SUCCESS] Vdisk myvdisk01's DTL is enabled and running.
[INFO] Recap of volumedriver check_dtl!
[INFO] ======================
[INFO] SUCCESS=1 FAILED=0 SKIPPED=0 WARNING=0 EXCEPTION=0
root@ovs-node-1:~# 
```

### Test result
Test passed.

### Packages
```
- openvstorage                         2.7.5-rev.4281.6612d8c-1        amd64        openvStorage
- openvstorage-backend                 1.7.5-rev.811.c1b83e6-1         amd64        openvStorage Backend plugin
- openvstorage-backend-core            1.7.5-rev.811.c1b83e6-1         amd64        openvStorage Backend plugin core
- openvstorage-backend-webapps         1.7.5-rev.811.c1b83e6-1         amd64        openvStorage Backend plugin Web Applications
- openvstorage-core                    2.7.5-rev.4281.6612d8c-1        amd64        openvStorage core
- openvstorage-hc                      1.7.5-rev.811.c1b83e6-1         amd64        openvStorage Backend plugin HyperConverged
- openvstorage-health-check            3.1.2-rev.209.3f31a72-1         amd64        Open vStorage HealthCheck
- openvstorage-sdm                     1.6.5-rev.423.713f250-1         amd64        Open vStorage Backend ASD Manager
- openvstorage-webapps                 2.7.5-rev.4281.6612d8c-1        amd64        openvStorage Web Applications
```
